system-parachains/people/people-polkadot/src/people.rs

```diff
@@ -0,0 +1,229 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::*;
+use crate::xcm_config::LocationToAccountId;
+use codec::{Decode, Encode, MaxEncodedLen};
+use enumflags2::{bitflags, BitFlags};
+use frame_support::{
+	parameter_types, CloneNoBound, EqNoBound, PartialEqNoBound, RuntimeDebugNoBound,
+};
+use pallet_identity::{Data, IdentityInformationProvider};
+use parachains_common::{impls::ToParentTreasury, DAYS};
+use scale_info::TypeInfo;
+use sp_runtime::{
+	traits::{AccountIdConversion, Verify},
+	RuntimeDebug,
+};
+use sp_std::prelude::*;
+
+parameter_types! {
+	//   27 | Min encoded size of `Registration`
+	// - 10 | Min encoded size of `IdentityInfo`
+	// -----|
+	//   17 | Min size without `IdentityInfo` (accounted for in byte deposit)
+	pub const BasicDeposit: Balance = system_para_deposit(1, 17);
+	pub const ByteDeposit: Balance = system_para_deposit(0, 1);
+	pub const SubAccountDeposit: Balance = system_para_deposit(1, 53);
+	pub RelayTreasuryAccount: AccountId =
+		parachains_common::TREASURY_PALLET_ID.into_account_truncating();
+}
+
+impl pallet_identity::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type Currency = Balances;
+	type BasicDeposit = BasicDeposit;
+	type ByteDeposit = ByteDeposit;
+	type SubAccountDeposit = SubAccountDeposit;
+	type MaxSubAccounts = ConstU32<100>;
+	type IdentityInformation = IdentityInfo;
+	type MaxRegistrars = ConstU32<20>;
+	type Slashed = ToParentTreasury<RelayTreasuryAccount, LocationToAccountId, Runtime>;
+	type ForceOrigin = EnsureRoot<Self::AccountId>;
+	type RegistrarOrigin = EnsureRoot<Self::AccountId>;
+	type OffchainSignature = Signature;
+	type SigningPublicKey = <Signature as Verify>::Signer;
+	type UsernameAuthorityOrigin = EnsureRoot<Self::AccountId>;
+	type PendingUsernameExpiration = ConstU32<{ 7 * DAYS }>;
+	type MaxSuffixLength = ConstU32<7>;
+	type MaxUsernameLength = ConstU32<32>;
+	type WeightInfo = weights::pallet_identity::WeightInfo<Runtime>;
+}
+
+/// The fields that we use to identify the owner of an account with. Each corresponds to a field
+/// in the `IdentityInfo` struct.
+#[bitflags]
+#[repr(u64)]
+#[derive(Clone, Copy, PartialEq, Eq, RuntimeDebug)]
+pub enum IdentityField {
```

Comment:
this can be shared with the kusama people chain? unless the kusama people chain is using a different setup for some reason

Reply:
it can be in a sub module `system-parachains/common`. but I wont move it to polkadot-sdk

Reply:
I disagree, IMO it should not be shared. For one, we prioritized things like `GitHub` and `Discord` based on the Fellowship/Ambassador Program wanting these to be first-class fields. The Kusama community could very well could request special fields (if they decide to keep the chain, see following). Two, [as you requested](https://github.com/polkadot-fellows/RFCs/pull/32#discussion_r1343381286), I would propose removing the Kusama chain/info after Polkadot's is successfully launched. This is not much code and I don't think it's a big deal to have this bit of the config copied, but if more people are against it then sure it can move.

Reply:
if we are going to remove kusama people chain soonish then this is fine